public/js/main.js

```diff
@@ -87,7 +87,7 @@ HacktoberfestChecker.prototype.getUsernameIssues = function(e) {
 HacktoberfestChecker.prototype.usernameIssuesSuccess = function(html, textStatus, xhr) {
     this.results.html(html);
     window.twttr.widgets.load();
-
+    window.FB.XFBML.parse();
```

Comment:
Can we add a newline after. Alos may be nice to put this and the twitter one in a `loadSocialWidgets()` method or something?

Reply:
sure, one moment)